understanding/20/headings-and-labels.html

```diff
@@ -31,25 +31,23 @@ <h2>Intent of Headings and Labels</h2>
          may suffice if it provides an appropriate cue to finding and navigating content.
       </p>
       
-      <p>This Success Criterion does not require headings or labels. This Success Criterion
+      <p>This Success Criterion does not require headings or labels. (Labels for inputs are covered separately by <a href="labels-or-instructions">3.3.2 Labels or Instructions</a>). This Success Criterion
```

Comment:
maybe a small tweak - I'm not a super-fan of this parenthetical crammed in here. I might suggest that instead of here, this is moved after the next sentence, not in parentheses, and expanded a bit more along the lines of "The need for inputs to have labels is covered separately by ..."

Reply:
agree, it's awkward either way, but yeah inverting the sentence could also work.

Reply:
I incorporated the changes discussed in TF call into new commit, so marking this conversation as resolved.